microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md

```diff
@@ -35,21 +35,22 @@ An overview of the changes involved, as described in [Migrate to Manifest V3](ht
 * Various minor feature improvements.
 
 
-<!-- ---------------------------------------------------------------------- -->
+<!-- ====================================================================== -->
 ## Manifest timeline for the Chromium browser engine
 
-Chromium has revised the timelines for ending support for Manifest V2.  See:
+Chromium has revised the timelines for ending support for Manifest V2.<!-- todo: add "in/for Chrome"? -->  See:
 
 * [Manifest V2 support timeline](https://developer.chrome.com/docs/extensions/develop/migrate/mv2-deprecation-timeline)
 * [The phase-out timeline](https://developer.chrome.com/blog/resuming-the-transition-to-mv3#the_phase-out_timeline) in _Resuming the transition to Manifest V3_.
 
 
-<!-- ---------------------------------------------------------------------- -->
+<!-- ====================================================================== -->
 ## Manifest timeline for Microsoft Edge and Partner Center
+<!-- todo: condense this h2 section a lot -->
```

Comment:
condense this h2 section a lot - move all the takeaway summary points into the table
@sivMSFT